phases/ephemeral/witx/typenames.witx

```diff
@@ -1,767 +1,767 @@
-;; Type names used by low-level WASI interfaces.
+;;; Type names used by low-level WASI interfaces.
 ;;
```

Comment:
To confirm, I think the intention is that this is `;;;`, right? (perhaps just a sed script gone wrong?)

Reply:
ah, yeah, I missed that the very top comment on each file is not a doc. fixed.